src/docs/santiment-queries/liquidation/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Liquidation Overview
+author: Santiment Team
+date: 2023-04-03
+description: Information about the Liquidation project's tables stored in Santiment Queries.
+---
+
+## Overview
+
+Liquidation is a project that aims to track deposits, borrows, repayments, withdrawals, and liquidation actions occurring on lending protocols.
+
+In **Santiment Queries**, we collect liquidation data from the following platforms:
+
+- AAVE-v2/v3
+- Compound
+- Liquity
+- MakerDAO
+
+## Usage
+
+The Liquidation project can be used for the following purposes:
+
+- Monitor the inflow and outflow of lending protocols
+- Observe liquidation actions occurring on each protocol
+- Track the Total Value Locked (TVL) in various protocols
+- Determine the total amount borrowed for a specific asset
+
+## Exploration of the Tables
+
+We can inspect the tables with the commands we learned in [Exploration](/santiment-queries/exploration/).
+
+Below, we have listed all the liquidation tables with their columns, brief descriptions, and some example queries that you can use to explore the tables.
```

Comment:
This sentence should be reworked because there is a single table and not many tables